File: Python/Patterns/squarestar/looper1.py
row = int(input("Enter number of rows: "))
col = int(input("Enter number of columns: "))

original_col = col  # Save the original value

while row > 0:
    temp_col = original_col  # Reset column counter for each row
    while temp_col > 0:
        print("  *", end="")
        temp_col -= 1
    print()  # Move to the next line
    row -= 1

File: CPP/Patterns/squarestar/looper1.cpp
#include <iostream>
using namespace std;

int main() {
    int row, col;
    cout << "Enter the row: ";
    cin >> row;
    cout << "Enter the column: ";
    cin >> col;

    int i = 0;
    while (i < row) {
        int j = 0;  // Reset column counter for each row
        while (j < col) {
            cout << " *";
            j++;
        }
        cout << endl;
        i++;
    }

    return 0;
}
